source/native-plugins/midi-transpose.c

```diff
@@ -116,10 +114,8 @@ static void miditranspose_set_parameter_value(NativePluginHandle handle, uint32_
     {
         case 0:
             handlePtr->octaves = (int)value;
-            break;
```

Comment:
hmm this is wrong now. you need the break here as it does not have a return, continue or other function stopper.
without the break, the cases fall through each other.